suites/reef/rbd/tier-2_rbd_regression.yaml

```diff
@@ -199,3 +199,17 @@ tests:
       module: test_rbd_compression.py
       name: Test to verify data compression on global, pool and image level
       polarion-id: CEPH-83574644
+
+  - test:
```

Comment:
This test would be applicable to pacific and quincy, as well Is it?

Reply:
Added test to pacific and quincy suite as well.